### DISCUSSION TITLE
Search on "All GitHub" for some given string does not search for the actual string instead it finds a lot of arbitrary junk.

### DISCUSSION THREAD

**Discussion by jarlfr** [community/community#40866] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When you search for a string in scope "All GitHub", the so call "Best match" does not search first for the literal string you give. Instead it adds all kind of stupid things into your string. Why? Results are jumbled up with all kinds of nonsense.

**Comment by ossanna16** [comment]
Hi there @jarlfr and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
